src/com/team2502/robot2018/Robot.java

```diff
@@ -177,8 +185,17 @@ public void disabledPeriodic()
      */
     public void autonomousInit()
     {
+        NavXLocationEstimator rotEstimator = new NavXLocationEstimator();
+        EncoderDifferentialDriveLocationEstimator encoderDifferentialDriveLocationEstimator = new EncoderDifferentialDriveLocationEstimator(rotEstimator);
```

Comment:
Put as final static variables in either Robot or Constants

Reply:
Why? They aren't used anywhere else, so I don't see their purpose.

Reply:
Occasionally, for debugging, we may need to access them and ask them for our estimated pos/rotation.